datalad/support/tests/test_annexrepo.py

```diff
@@ -1617,30 +1618,32 @@ def test_AnnexRepo_metadata(path):
     # recursive needs a flag
     assert_raises(CommandError, ar.set_metadata, '.', purge=['virgin'])
     ar.set_metadata('.', purge=['virgin'], recursive=True)
-    eq_({'up.dat': {
+    deq_({'up.dat': {
         'mike': ['awesome']}},
         ar.get_metadata('up.dat'))
-    ar.set_metadata('.', reset={'tag': 'one'}, purge=['mike'], recursive=True)
+    # Use trickier tags (spaces, =)
+    ar.set_metadata('.', reset={'tag': 'one and= '}, purge=['mike'], recursive=True)
     playfile = opj('d o"w n', 'd o w n.dat')
     target = {
         'up.dat': {
-            'tag': ['one']},
+            'tag': ['one and= ']},
```

Comment:
not to myself -- here using name "tag" is similar to using `annex metadata --tag`/`--untag`, so we do not need extending interface. nicely done @mih